front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -120,11 +121,16 @@ const StdcmViewV2 = () => {
     }
   }, [simulationsList]);
 
+  // If we've got an error during the loading of the stdmc env which is not the "no config error" message,
+  // we let the error boundary manage it
+  if (error && error.message !== NO_CONFIG_FOUND_MSG) throw error;
+
   return (
     <div role="button" tabIndex={0} className="stdcm-v2" onClick={() => setShowStatusBanner(false)}>
       <StdcmHeader />
-      <div>
-        {scenarioID && (
+
+      {isNil(error) ? (
```

Comment:
I think it would be easier to read if you write it like this: 
```
if error
  then display error
else
  display the body
```
because the errorComponent is way smaller than the main component.